frontend/shared/components/src/events/EditEventView.vue

```diff
@@ -133,12 +131,12 @@
             <h2 class="style-with-button">
                 <div>Regio</div>
                 <div>
-                    <button type="button" class="button icon trash" @click="deleteTagRestriction" />
+                    <button v-if="!hasTagRestrictions" type="button" class="button icon trash" @click="deleteTagRestriction" />
                 </div>
             </h2>
             <p>Kies voor welke groepen deze activiteit zichtbaar is.</p>
 
-            <TagIdsInput v-model="organizationTagIds" />
+            <TagIdsInput v-model="organizationTagIds" :is-tag-enabled-operator="isTagEnabledOperator" />
```

Comment:
```suggestion
            <TagIdsInput v-model="organizationTagIds" :is-tag-enabled="isTagEnabledOperator" />
```
Het woordje 'operator' maakt het misschien complexer dan het is, waardoor het moeilijker is om te begrijpen wat dit betekent